teachertool/src/components/DebugInput.tsx

```diff
@@ -1,27 +1,19 @@
+/// <reference path="../../../built/pxtblocks.d.ts"/>
+
 import { useState } from "react";
 import { Button } from "react-common/components/controls/Button";
 import { Input } from "react-common/components/controls/Input";
 import { Textarea } from "react-common/components/controls/Textarea";
-import { getProjectMetaAsync, getProjectTextAsync } from "../services/BackendRequests";
+import { runEvaluateAsync } from "../transforms/runEvaluate";
```

Comment:
Can you please rename the file to match the function name? `runEvaluateAsync.ts`